api/pom.xml

```diff
@@ -24,15 +24,18 @@
         <!-- This is just for now and will not work if the API has a separate release cycle than the rest. -->
         <groupId>org.eclipse.microprofile.reactive-streams-operators</groupId>
         <artifactId>microprofile-reactive-streams-operators-parent</artifactId>
-        <version>2.1-SNAPSHOT</version>
+        <version>3.0-SNAPSHOT</version>
```

Comment:
I took a further look at the changes. It seems that the spec has no dependency at all on Jakarta specs. However, it is still good to go through this exercise to fix up license and adopt microprofile parent pom.xml.
However, the spec version does not need to be a major change. A minor or micro change is sufficient. I think in light of this exercise, just do a minor change. Please correct me if this is not the case.
I suggest we don't change this at all as the release will automatically update it to 2.2-SNAPSHOT once 2.1 is released. Please undo the version changes in other places.

Reply:
You are right package change impacted only the tck classes

Reply:
Done